{"name": "测试用例描述：错误的账号登录失败", "status": "passed", "description": "测试描述：错误的账号登录失败", "steps": [{"name": "输入错误的账号", "status": "passed", "attachments": [{"name": "错误的账号", "source": "99dad56f-8d12-4c0f-9b03-d76dccc1fcdf-attachment.attach"}], "start": 1651987630292, "stop": 1651987630294}], "attachments": [{"name": "截图", "source": "9a8c2246-1928-420a-b9c1-ce567ec2001b-attachment.png", "type": "image/png"}, {"name": "截图", "source": "d5f0b568-ab92-426c-ada1-8d618aabc651-attachment.png", "type": "image/png"}], "start": 1651987630292, "stop": 1651987634652, "uuid": "cfb0c05b-60ca-44cb-98c9-4bc210837123", "historyId": "07ccc934c01f5e078d85f2f38ccbb5fd", "testCaseId": "07642125497ce5d4c888927c76cac843", "fullName": "test_two.Test#test_21", "labels": [{"name": "epic", "value": "xxx测试"}, {"name": "severity", "value": "critical"}, {"name": "story", "value": "登录"}, {"name": "feature", "value": "登录模块"}, {"name": "suite", "value": "test_two"}, {"name": "subSuite", "value": "Test"}, {"name": "host", "value": "MS-SLSXLICKSUSF"}, {"name": "thread", "value": "16428-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_two"}]}